pom.xml

```diff
@@ -313,6 +313,16 @@
                 </plugin>
             </plugins>
         </pluginManagement>
+        <plugins>
```

Comment:
Do we need this?

Reply:
It is required for java 11 features  but  if it creates problem I can use another instruction
FileSystemIndexTransferer -> line 52
        Path destination = Path.of(destDir);